README.md

```diff
@@ -162,225 +189,414 @@ and modify the `system-reserved` line under `kubeletExtraArgs`:
 ./deploy-test-resources.sh
 ```
 
-5. :gear: If Konflux was installed on a cluster hosted in a remote machine, SSH port-forwarding can 
-be used to access. Open an additional terminal and run the following command
-(make sure to add the details of your remote machine and user):
+5. :gear: As a reminder, if you've installed Konflux on a remote host, you will need to configure SSH port
+forwarding to access your cluster. To do this, open an additional terminal and run the following command
+(make sure to add the details for your remote user and IP address):
 
 ```bash
-ssh -L 9443:localhost:9443 $USER@$VM_IP
+ssh -L 9443:localhost:9443 <remote_host_user_name>@<remote_host_ip_address>
```

Comment:
I think the changes here are fine. I don't think we should include the command at the top of the document, as it may cause confusion there and also become a maintenance burden.